src/components/comments/Answer.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+export default function Answer(props) {
+  return (
+    <div style={styles.answerContainer}>
+      <div style={styles.answerHeaderStyle}>{props.answerHeaderText}</div>
+      <div style={styles.answerStyle}>{props.answerText}</div>
+    </div>
+  );
+}
+
+const styles = {
+  answerContainer: {
+    backgroundColor: "#EEF0F1",
+    borderRadius: "25px",
+    marginLeft: "200px",
+    marginRight: "10px",
+    padding: "10px",
+  },
+  answerHeaderStyle: {
+    textAlign: "left",
+    color: "black",
+    fontFamily: "Mulish",
```

Comment:
If you want to use a custom font like `Mulish`, you'll need to import the font somewhere in your app, either by:

1. use an external font CDN, like[ Google Fonts](https://fonts.google.com/specimen/Poppins?query=poppi&sidebar.open=true&selection.family=Poppins:wght@500), and put the <link> tag in your public/index.html file, with the relevant @font-face declaration in index.css. This is extremely convenient, and use benefits of browser-based caching (e.g. maybe they already have hit the CDN because they’ve been on the ACM Website), but has the downside of creating an extra (blocking) network request and some offline shenanigans.
2. use a package like[ typefaces](https://github.com/KyleAMathews/typefaces), which works out of-the-box with CRA and lets NPM/webpack deal with bundling your font in your final production package. It’s still relatively convenient, and has the benefit of reducing network requests (compared to option 1), but doesn’t take advantage of your browser cache.

Reply:
Keep in mind you can also use classes or set a global font style if that's what you want.

Reply:
That was just a placeholder of a font that we found cool on Figma, we still have to decide which font we want to go for in the long run